Anagrams/src/ua/goit/alg/TestAnagrams.java

```diff
@@ -0,0 +1,43 @@
+package ua.goit.alg;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.InputStream;
+import java.io.PrintStream;
+import java.io.UnsupportedEncodingException;
+import org.junit.Test;
+
+public class TestAnagrams {
+
+    @Test
+    public void twoWords() throws UnsupportedEncodingException {
+        String input = "hello world!";
+        String expected = " olleh !dlrow";
+        InputStream stubInputStream = new ByteArrayInputStream(input.getBytes("UTF-8"));
```

Comment:
refactor this BAIS thingy into the outputToString method, it's duplicated all over